docker-README.md

```diff
@@ -100,25 +100,26 @@ has to be provided in  `node.[name].ssh.*` properties.
+| Property name             | Description                                                       | Default value                      |
+|:--------------------------|:------------------------------------------------------------------|:-----------------------------------|
+| nodegroup                 | Name of the node group for this node. Default value should typically be satisfactory. | The `nodegroup` value from the cloud provider. |
+| docker.image              | Image name from which the node is started                         |                                    |
+| docker.bootScript         | Allows you to specify a script that is to be run on boot. The script is run with `sudo`. | [None. Optional.] |
+| docker.bootScript.file    | As `docker.bootScript`, but allows you to specify a path to a file that contains the script. Only one of `docker.bootScript` and `docker.bootScript.file` can be specified at a time. | [None. Optional.] |
+| docker.networkMode        | NetworkMode used                                                  | `host`                             |
+| docker.capAdd             | Comma separated list of Linux capabilities to be added.           |                                    |
+| docker.cmd                | Docker command                                                    |                                    |
+| docker.entrypoint         | Docker entrypoint | |
+| docker.env                | Environment variables in form `KEY=VALUE` separated by value defined in `docker.env.splitter` |        |
+| docker.env.splitter       | Separator RegEx for `docker.env` values                           |  `,`                               |
+| docker.inboundPorts       | which ports should be open                                        |                                    |
+| docker.portBindings       | Comma separated list of TCP port bindings in form "hostPort:nodePort" |                                |
+| docker.privileged         | true/false flag which controls if the container is privileged     | false                              |
+| docker.ssh.port           | SSH port number if SSH is installed (not-mapped - i.e. in-container value) |                           |
+| docker.ssh.user           | SSH username                                                      |                                    |
+| docker.ssh.password       | SSH user password                                                 |                                    |
+| docker.ssh.privateKey     | The SSH private key (not the file of the key!). In case you want to pass a private key manually. Currently unused in favor of `docker.ssh.privateKeyFile`. | [None. Optional.] |
+| docker.ssh.privateKeyFile | The path to the private key which should be used to connect to the instance. Necessary for establishing an SSH channel in your tests. | [None. Optional.] |
```

Comment:
I think we should either document that `docker.ssh.privateKeyFile` overrides `docker.ssh.privateKey` or check whether both are supplied, and if yes, fail.